core/vm.cpp

```diff
@@ -881,8 +921,18 @@ class Interpreter {
         builtins["parseJson"] = &Interpreter::builtinParseJson;
         builtins["encodeUTF8"] = &Interpreter::builtinEncodeUTF8;
         builtins["decodeUTF8"] = &Interpreter::builtinDecodeUTF8;
+
+        DesugaredObject *stdlib = makeStdlibAST(alloc, "__internal__");
+        jsonnet_static_analysis(stdlib);
+        stdlibAST = stdlib; // stdlibAST is const, so we need to do analysis before this assignment
+        stdThunk = makeHeap<HeapThunk>(nullptr, nullptr, 0, static_cast<const AST*>(stdlibAST));
```

Comment:
Can stdThunk be a local var instead of a field?  And is it possible to do it without the thunk at all?

Reply:
I think if you let the context be null in the newCall, the worst that will happen is a slightly worse stacktrace if there was a problem evaluating the stdlib (which is a simple file containing only an object literal anyway).